--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -404,6 +404,7 @@ define KernelPackage/sound-hda-codec-realtek
   TITLE:= HD Audio Realtek Codec
   KCONFIG:= \
 	CONFIG_SND_HDA_CODEC_REALTEK
   FILES:= \
 	$(LINUX_DIR)/sound/pci/hda/snd-hda-codec-realtek.ko@lt6.18 \
 	$(LINUX_DIR)/sound/hda/codecs/realtek/snd-hda-codec-realtek-lib.ko@ge6.18
+  DEPENDS+=@!LINUX_6_18
   AUTOLOAD:=$(call AutoProbe,snd-hda-codec-realtek LINUX_6_18:snd-hda-codec-realtek-lib)
   $(call AddDepends/sound,kmod-sound-hda-core +kmod-snd-hda-scodec-component)
 endef
